test/board_test.rb

```diff
@@ -64,22 +66,21 @@ def test_it_can_validate_placement_if_coordinates_are_diagonal
   end
 
   def test_it_can_place_a_ship
-    skip
     @board.place(@cruiser, ["A1", "A2", "A3"])
-    cell_1 = board.cells["A1"]
-    cell_2 = board.cells["A2"]
-    cell_3 = board.cells["A3"]
-    assert_equal @cruiser, cell_1.ship
-    assert_equal @cruiser, cell_2.ship
-    assert_equal @cruiser, cell_3.ship
-    assert_equal true, cell_3.ship == cell_2.ship
+    # cell_1 = board.cells["A1"]
+    # cell_2 = board.cells["A2"]
+    # cell_3 = board.cells["A3"]
+    # require 'pry' ; binding.pry
+    assert_equal @cruiser, @board.cells["A1"].ship
+    assert_equal @cruiser, @board.cells["A2"].ship
+    assert_equal @cruiser, @board.cells["A3"].ship
+    assert @board.cells["A1"].ship == @board.cells["A2"].ship
   end
 
   def test_it_will_not_overlap_when_ship_placed
-    skip
     @board.place(@cruiser, ["A1", "A2", "A3"])
-    expected = @board.valid_placement?(@submarine, ["A1", "B1"])
-    assert_equal false, expected
+    binding.pry
+    refute @board.valid_placement?(@submarine, ["A1", "B1"])
   end
```

Comment:
```suggestion
  expected = @board.valid_placement?(@submarine, ["A1", "B1"])
  refute_equal false, expected
```
Since any falsy value could pass this test, explicitly expecting false would be safer. What do you think?